src/reanimated2/shareableCache.ts

```diff
@@ -12,4 +23,22 @@ There we use Object.entries to iterate over the keys which throws an error on ac
 For convenience we move this cache to a separate file so it doesn't scare us with red squiggles.
 */
 
-export const shareableCache = new WeakMap<object, ShareableRef | symbol>();
+const cache = SHOULD_BE_USE_WEB
+  ? null
+  : new WeakMap<object, ShareableRef | symbol>();
+
+export const shareableCache = cache
```

Comment:
This is more of a philosophical decision but I think we should use base conditions instead of previous effects. This leads to duplicated code (in some sense) but probably easier to understand.
```suggestion
export const shareableCache = SHOULD_BE_USE_WEB
```

Reply:
I wanted to do it, but you know... TypeScript...
but this is the right thing to do, since checking for `cache` here might not be a scalable solution.